weave-and-docker-platform/1-machine.md

```diff
@@ -1,150 +1,233 @@
+
+Specifically, you will:
+
+  1. Install Docker Machine and Weave
+  2. Setup Weave onto a single VM on VirtualBox
+  3. Deploy a basic _"Hello, Weave!"_ application
+  4. Enable {{ weavedns }} to discover the Weave-attached Docker containers on the network.
+  5. Communicate with your app and send a message from one container to another using the Docker API {{ weaveproxy }}.
+
+This tutorial uses very simple UNIX tools, and it doesn't require any programming skills.
+
+This tutorial will take about 10 minutes to complete.
 
 ## What you will use
 
   - [Weave](http://weave.works)
-  - [Docker & Machine](http://docker.com)
+  - [Docker & Docker Machine](https://docs.docker.com)
+
+## What You Need to Complete Part 1
 
-## What you will need to complete this chapter
+If you are using OS X or Windows, you can install [Docker Toolbox](https://www.docker.com/toolbox), which provides all the tools you will need.
+
+For other operating systems, you will need to install and configure the following separately before proceeding:
 
-  - 10-15 minutes
   - [`docker-machine`](http://docs.docker.com/machine/#installation) binary (_`>= 0.2.0`_)
   - [`docker`](https://docs.docker.com/installation/#installation) binary, at lest the client (_`>= v1.6.x`_)
   - [VirtualBox](https://www.virtualbox.org/wiki/Downloads) (_`>= 4.3.x`_)
   - `curl` (_any version_)
 
-If you are using OS X, then you can install these tools with Homebrew like this:
+## Let's go!
+
+First, install the `weave` command. Note that if you don't have ownership of the `/usr/local/bin` directory, you may need to preface these commands with `sudo`:
 
-    brew install docker docker-machine
+~~~bash
+curl -L git.io/weave -o /usr/local/bin/weave
+chmod a+x /usr/local/bin/weave
+~~~
 
-For other operating systems, please refer to links above.
+Next create a Virtual Machine or a VM on the VirtualBox, called `weave-1`, by running:
 
-If you haven't yet installed VirtualBox, be sure to follow [installation instructions for your OS](https://www.virtualbox.org/wiki/Downloads).
+~~~bash
+docker-machine create -d virtualbox weave-1
+~~~
 
-## Let's go!
+Once the VM is running, configure your shell environment by running:
+
+~~~bash
+eval "$(docker-machine config weave-1)"
+~~~
+
+Verify that everything installed correctly:
+
+~~~bash
+docker info
+~~~
+
+Now launch the Weave network and set up the proxy:
 
-### Launch
+~~~bash
+weave launch
+~~~
 
-First, you will need to obtain Weave script.
 
-    curl -OL git.io/weave
-    chmod +x ./weave
+Next set up the weave environment for the Docker API proxy:
 
-Now, you should create a VM with
+~~~bash
+eval "$(weave env)"
+~~~
 
-    docker-machine create -d virtualbox weave-1
+Check to see that the proxy is running properly:
 
-Once the VM is up, you need to point Weave at this VM by setting `DOCKER_CLIENT_ARGS` environment variable.
+~~~bash
+docker logs weaveproxy
```

Comment:
I see where you are looking.  I thought one of changes in Weave was that the proxy now automatically starts on weave launch, so that you don't have to start it independently, except on OSX where the TLS settings are necessary. Is this not the case? 

https://github.com/weaveworks/weave/blob/master/site/proxy.md